integration-tests/smoke/vrfv2plus_test.go

```diff
@@ -2083,3 +2080,287 @@ func TestVRFv2PlusNodeReorg(t *testing.T) {
 	})
 
 }
+
+func TestVRFv2PlusBatchFulfillmentEnabledDisabled(t *testing.T) {
```

Comment:
well, tbh I would extract common logic to one function and then just pass as arguments vrfv2 and vrfv2plus specific parts so we don't have almost identical test